api/v1/cluster_types.go

```diff
@@ -135,6 +135,13 @@ type ClusterSpec struct {
 	// +optional
 	ImagePullPolicy corev1.PullPolicy `json:"imagePullPolicy,omitempty"`
 
+	// Name for custom sheduler when the default one cannot be used
+	// If not defined, it defaults to `default-scheduler`.
+	// More info: https://kubernetes.io/docs/concepts/scheduling-eviction/kube-scheduler/
+	// +optional
+	// +kubebuilder:default:=none
```

Comment:
```suggestion
	// Name for custom scheduler when the default one cannot be used.
	// More info: https://kubernetes.io/docs/concepts/scheduling-eviction/kube-scheduler/
	// +optional
```

Reply:
this will need to regenerate docs and CRDs

Reply:
I modified the description as suggested and regenerated the docs and CRDs.